data/hai/hai missions.txt

```diff
@@ -209,6 +209,69 @@ mission "Ask the Hai about the Unfettered"
+					goto acknowledge
+				`	"I think you have the wrong captain."`
+			`	He slaps a device on the table and a small holographic projection of your face springs up and begins rotating around. "No, I think not."`
+			label acknowledge
+			`	He heaves a sigh. "You've been fighting the Unfettered you see, and there are some Hai that don't take too kindly to that. They view you as an outsider sticking your nose into Hai internal politics. Some of those are what we merchants like to call Sympathizers; and they can get a little carried away with themselves and fire off a few shots if they see someone like yourself getting a little too gun-happy where the Unfettered are concerned."`
+			`	With a wave of his hand he indicates the spaceport around you. "You might have noticed some difficulty landing potentially. You see because there are lots of people who don't like outsiders putting their noses in where they're not wanted, if you happen to shoot back and accidentally destroy a Sympathizer's ship, the authorities don't much care who shot first. There are enough of those opinions to ensure that if you shoot down a Sympathizer, it is treated no differently than if you shot down any other Hai merchant. The security forces will treat it the same.`
+			`	"Now the merchants, we're a different matter. It is a topic of great and ongoing debate among us, so you might notice that some people are less offended when you shoot down someone they've been arguing with, but everyone has friends on all sides, so you should watch yourself nonetheless. Sympathizers may become upset enough to shoot at you anywhere they see you. Despite that, the simple fact of the matter is that, if you retaliate against a Hai, any Hai, it'll upset someone and count against you. Even if that Hai was a Sympathizer, and even if they shot first..."`
+			`	He looks at you very seriously to make sure what he's said has sunk in.`
+			choice
+				`	"I understand, I've stepped on some toes."`
+					goto trouble
+					to display
+						"reputation: Hai" < 1
+				`	"I understand, thank you for warning me."`
+					goto end
+					to display
+						"reputation: Hai" >= 1
+				`	"Why should I get in trouble for defending myself against Sympathizers."`
+			`	He shrugs. "You should try not to impart human ethical standards onto Hai. The simple fact of the matter is that they are citizens, and you are not.`
+			`	"When a report gets made that you have destroyed the vessel of a Hai citizen, it is up to the local security staff to verify the chain of events. If they report it was your fault, then it was your fault, unless you can challenge that in a court. As you are not a citizen though, the security apparatus can simply brand you as a pirate and prevent you from ever gaining access to a court to challenge it in the first place. Even if you managed to do so, you're a Captain; your business is in space. You would be grounded for months to work through the court proceedings to challenge it.`
```

Comment:
maybe this could be nuanced that you're not a citizen yet?
or do you really mean that they are this kind of society? because it feels like this could lead to abuse with humans born in Hai space or merchants who came there to live?

Reply:
As a space-based captain, you're not a citizen full-stop.
They are this kind of society, but there are pathways to citizenship; they just require you to sit still and own something that gives you a vested interest in their society.

This criteria never applies to the player.

Reply:
good to know!
maybe that could be hinted somehow

Reply:
It's kind of alluded to, but there'll be more things that touch on that later too.
For the purpose of this, you're not a citizen right now, so the specifics about citizenship don't really belong here. It'd distract from the core thrust of this.